### PR TITLE
Spelling and URL fixes

### PR DESCRIPTION
**Pull request by modery**
Correcting some spellings and broken URLs

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/extend-and-develop.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Nearly every version of SharePoint has had a way of creating custom layouts for
...
+Microsoft recommends using [Power Automate](https://flow.microsoft.com) for configuring and executing all workflows in your Microsoft 365 environment, including SharePoint. For example, it&#39;s possible to create [unique approval workflows](/power-automate/modern-approvals) for content stored in SharePoint. Additionally, it&#39;s possible to use Power Automate as the default workflow engine [for approving SharePoint page content](https://support.office.com/article/a8b2e689-d4a1-4639-8028-333c0ece30d9), directly from the SharePoint user interface. Flows can be [triggered by SharePoint](/connectors/sharepointonline/#triggers) actions (such as when an item is created in a list), or perform [actions within SharePoint](/connectors/sharepointonline/#actions) (such as update a list item). While SharePoint Designer workflows are still supported, new workflows should be created using Power Automate.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/extend-and-develop.md:43]:
```suggestion
We recommend using [Power Automate](https://flow.microsoft.com) for configuring and executing all workflows in your Microsoft 365 environment, including SharePoint. For example, it&#39;s possible to create [unique approval workflows](/power-automate/modern-approvals) for content stored in SharePoint. Additionally, it&#39;s possible to use Power Automate as the default workflow engine [for approving SharePoint page content](https://support.office.com/article/a8b2e689-d4a1-4639-8028-333c0ece30d9), directly from the SharePoint user interface. Flows can be [triggered by SharePoint](/connectors/sharepointonline/#triggers) actions (such as when an item is created in a list), or perform [actions within SharePoint](/connectors/sharepointonline/#actions) (such as update a list item). While SharePoint Designer workflows are still supported, new workflows should be created using Power Automate.
```

File: SharePoint/SharePointServer/install/multiple-servers-for-a-three-tier-farm.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ Before you begin to install and configure SharePoint 2013, do the following:
...
+> The Distributed Cache service gives you a complete social computing experience. For more information about the Distributed Cache service, see [Overview of microblog features, feeds, and the Distributed Cache service in SharePoint Server](../administration/administration.md), [Manage the Distributed Cache service in SharePoint Server](../administration/manage-the-distributed-cache-service.md), [Plan for feeds and the Distributed Cache service in SharePoint Server](../administration/plan-for-feeds-and-the-distributed-cache-service.md), and [What's new in authentication for SharePoint Server 2013](../what-s-new/new-and-improved-features-in-sharepoint-server-2016.md)
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/install/multiple-servers-for-a-three-tier-farm.md:83]:
```suggestion
> The Distributed Cache service gives you a complete social computing experience. For more information about the Distributed Cache service, see [Overview of microblog features, feeds, and the Distributed Cache service in SharePoint Server](../administration/administration.md), [Manage the Distributed Cache service in SharePoint Server](../administration/manage-the-distributed-cache-service.md), [Plan for feeds and the Distributed Cache service in SharePoint Server](../administration/plan-for-feeds-and-the-distributed-cache-service.md), and [What's new in authentication for SharePoint Server 2013](../what-s-new/new-and-improved-features-in-sharepoint-server-2016.md).
```

File: SharePoint/SharePointServer/install/software-requirements-for-database-servers-for-sharepoint-server-subscription-edition.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ SharePoint server supports the following database versions:
 
 - A Standard or Enterprise Edition of SQL Server for Windows that supports database compatibility level 150. This includes SQL Server 2019 and any future version of SQL Server for Windows that supports database compatibility level 150. For more information about database compatibility levels, see [Compatibility Certification](https://docs.microsoft.com/sql/database-engine/install-windows/compatibility-certification?view=sql-server-ver15) and [ALTER DATABASE (Transact-SQL) Compatibility Level](https://docs.microsoft.com/sql/t-sql/statements/alter-database-transact-sql-compatibility-level?view=sql-server-ver15).
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/install/software-requirements-for-database-servers-for-sharepoint-server-subscription-edition.md:38]:
```suggestion
- A Standard or Enterprise Edition of SQL Server for Windows that supports database compatibility level 150. This includes SQL Server 2019 and any future version of SQL Server for Windows that supports database compatibility level 150. For more information about database compatibility levels, see [Compatibility Certification](/sql/database-engine/install-windows/compatibility-certification?view=sql-server-ver15) and [ALTER DATABASE (Transact-SQL) Compatibility Level](/sql/t-sql/statements/alter-database-transact-sql-compatibility-level?view=sql-server-ver15).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @modery, kindly commit the few additional suggested changes if you agree with them. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**